internal/eval/commands.go

```diff
@@ -1206,6 +1206,45 @@ var (
 		Arity:    3,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	linsertCmdMeta = DiceCmdMeta{
+		Name: "LINSERT",
+		Info: `
+		Usage:
+			LINSERT key <BEFORE | AFTER> pivot element
+		Info:
+			Inserts element in the list stored at key either before or after the reference value pivot.
+			When key does not exist, it is considered an empty list and no operation is performed.
+			An error is returned when key exists but does not hold a list value.
+		Returns:
+			Integer - the list length after a successful insert operation.
+			0 when the key doesn't exist.
+			-1 when the pivot wasn't found.
+		`,
+		Eval:     evalLINSERT,
+		Arity:    4,
+		KeySpecs: KeySpecs{BeginIndex: 1},
+	}
+	lrangeCmdMeta = DiceCmdMeta{
+		Name: "LRANGE",
+		Info: `
+		Usage:
+			LRANGE key start stop
+		Info:
+			Returns the specified elements of the list stored at key.
+			The offsets start and stop are zero-based indexes, with 0 being the first element of the list (the head of the list), 1 being the next element and so on.
+
+			These offsets can also be negative numbers indicating offsets starting at the end of the list.
+			For example, -1 is the last element of the list, -2 the penultimate, and so on.
+			
+			Out of range indexes will not produce an error. If start is larger than the end of the list, an empty list is returned.
+			If stop is larger than the actual end of the list it will be treated like the last element of the list.
+		Returns:
+			Array reply: a list of elements in the specified range, or an empty array if the key doesn't exist.
+		`,
+		Eval:     evalLRANGE,
+		Arity:    3,
```

Comment:
as per redis command info it is 4, I guess they count the command name as well. 
```suggestion
		Arity:    4,
```

Reply:
Done.